_posts/blog/2020-03-16-chi-hack-night-going-remote.md

```diff
@@ -0,0 +1,48 @@
+description: "Are you already feeling that Tuesday night void? Looking for a way to help out and contribute during the COVID-19 outbreak? Chi Hack Nights are going remote! Same time, but now from the comfort of your own home / quarantine location!"
+date: 2020-03-16
+image: 
+author: Eric Sherman
+author_url:
+author_image: /images/people/eric_sherman.jpg
+author_bio: "Board Member, Chi Hack Night"
+author2: Derek Eder
+author2_url: https://twitter.com/derekeder
+author2_image: /images/people/derek_eder.jpg
+author2_bio: "President, Chi Hack Night"
+
+published: true
+---
+
+*Due to the ongoing outbreak of COVID-19 (Coronavirus), in-person [Chi Hack Night is Postponed Until Further Notice](/blog/2020/03/09/chi-hack-night-postponed.html)*
+
+Are you already feeling that Tuesday night void? Looking for a way to help out and contribute during the COVID-19 outbreak?
+
+Chi Hack Nights are going remote! Same time, but now from the comfort of your own home / quarantine location!
+
+Here’s how you can participate in Remote Chi Hack Nights:
+
+1. Join the Chi Hack Night Slack [(What is Slack?)](https://slack.com/help/articles/115004071768-What-is-Slack-): [http://slackme.chihacknight.org/](http://slackme.chihacknight.org/)
+2. Once you’re in, join the **#chihacknight-remote** channel. This will be the virtual hangout for Remote Chi Hack Night. Members of [Chi Hack Night’s Board](https://chihacknight.org/board-of-directors.html) will be around to answer questions, provide resources about working remotely, share memes, talk and create a positive space to hang out in. 
+3. Existing breakout groups can also pitch their projects in this channel. You can also start a new breakout group if you’d like! Want to learn more about starting a breakout group? [Read this informative blog post](https://chihacknight.org/blog/2020/02/28/so-you-want-to-start-a-breakout-group.html).
+
+As always, our [Code of Conduct](https://chihacknight.org/code-of-conduct.html) will apply to this new mode of hack nights. [Some basic etiquette](https://slackhq.com/etiquette-tips-in-slack) also goes a long way towards making the experience fun and useful for everyone!
+
+Do you have feedback, questions, concerns, or comments you'd like to share that aren't related to a particular breakout group? Feel free to share them by reaching out to [info@chihacknight.org](mailto:info@chihacknight.org). We know this is a new situation for everyone, and we are here to support our community in the best ways we can! 
+
+## Resources and actions to slow the spread of COVID-19 
+Chi Hack Night now has a **#covid19** channel you are welcome to join for sharing resources, info & opportunities to help specifically with this crisis. Here’s a few great examples we’ve shared out so far:
+
+**#FlattenTheCurve** A one-stop place with practical knowledge and tips on COVID-19, and what each of us can do to slow the spread of the virus:  [Flattenthecurve.com](https://www.flattenthecurve.com/)
```

Comment:
can these be ### instead of bold? Not sure exactly how that might affect the styling.

Reply:
### translates to an `h3` which puts them on their own line, so id rather not